### PR TITLE
Add payout settings to BVNK

### PR DESCRIPTION
**Pull request by XaosSintez**
## Summary

This PR adds settings to BVNK

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/GatewayAccountConfig/BVNK.yaml [hunk 1]
```diff
@@ -26,3 +26,13 @@ allOf:
           - merchantId
           - hawkAuthId
           - hawkAuthKey
+      settings:
+        type: object
+        description: BVNK settings object.
+        properties:
+          payoutCurrency:
+            type: string
+            description: Payout crypto currency.
```
Comment (nanobash) [inline openapi/components/schemas/GatewayAccountConfig/BVNK.yaml:35]: I think correct cryptocurrency is one word.

cc: @noon-dawg

```suggestion
            description: Payout cryptocurrency.
```

Comment (noon-dawg) [reply openapi/components/schemas/GatewayAccountConfig/BVNK.yaml:35]: Well spotted @nanobash!

**Review by justnero — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/BVNK.yaml [hunk 2]
```diff
@@ -26,3 +26,13 @@ allOf:
           - merchantId
           - hawkAuthId
           - hawkAuthKey
+      settings:
+        type: object
+        description: BVNK settings object.
+        properties:
+          payoutCurrency:
+            type: string
+            description: Payout cryptocurrency.
```
Comment (noon-dawg) [inline openapi/components/schemas/GatewayAccountConfig/BVNK.yaml:35]: suggestion: for readability
```suggestion
            description: Cryptocurrency of the payout.
```

File: openapi/components/schemas/GatewayAccountConfig/BVNK.yaml [hunk 3]
```diff
@@ -26,3 +26,13 @@ allOf:
...
+          payoutNetwork:
+            type: string
+            description: Blockchain network to be used.
```
Comment (noon-dawg) [inline openapi/components/schemas/GatewayAccountConfig/BVNK.yaml:38]: suggestion: for readability
```suggestion
            description: Blockchain network to use for the payout.
```

**Review by i-yasko — APPROVED** [review]

**Review by viceromag — APPROVED** [review]

**Pull request merged**